app/containers/activity/act.js

```diff
@@ -102,13 +103,14 @@ class Act extends Component {
       "devices:os":`devices:${DeviceInfo.getSystemName()}`,
       "devices:osversion":DeviceInfo.getSystemVersion(),
       "deviceModel":DeviceInfo.getModel(),
+      "appVersion": packageJson.name + packageJson.version,
```

Comment:
This is not correct app version. by doing so, we will have 3 different app versions. - package.json, ios, android

Reply:
@erik-ilyin can we just import the package.json version in iOS and Android to keep them in sync?

Reply:
Or do you have a suggested solution to just have one version number?

Reply:
no we can't import version in iOS and android. we should set version separate in iOS and android. However we can put same version name for display in IOS and Android and then import that version name here. Please remove these part for now. I will implement that later.

Reply:
> Please remove these part for now.

[Done](https://github.com/ChildMindInstitute/mindlogger-app/pull/51/commits/4a21abe2349ad22eb602596cc9bfb4f80cd7d042).